index.bs

```diff
@@ -2288,7 +2289,7 @@ Registering Formal Objections</h3>
 	related to that review.
 
 	Note: [=Formal Objections=] against matter in a [=technical report=]
-	are required to be addressed before <a href="#rec-advance">requesting advancement</a>
+	have to be addressed before <a href="#rec-advance">requesting advancement</a>
```

Comment:
is the 'must' aspect of this established elsewhere?

Reply:
Yes, in https://www.w3.org/2023/Process-20231103/#transition-reqs ; well, technically you can request the advancement, but the Team is required to withhold their approval in this case.